modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/java/JavaMethodSignature.java

```diff
@@ -19,14 +19,15 @@
 public class JavaMethodSignature {
 
 	public JavaMethodSignature(
-		String path, PathItem pathItem, Operation operation,
+		String path, PathItem pathItem, Operation operation, String operationId,
```

Comment:
Having operation and operationId is ambiguous. Can we have 2 constructors or simplify it in any other way?

Reply:
You are right. I updated with a couple of commits. Would you like to review? I found out there was an error autogenerating the Permissions endpoints by mistake, like the dinamite :clown_face: